### ISSUE TITLE
Channels.app - Unable To Scan Antenna

### ISSUE THREAD

**Issue by lswee** [fHDHR/fHDHR_plugin_interface_hdhr#17]
In [Channels](https://getchannels.com/).app (on AppleTV), I go to 'Add Tuner' and enter the IP:Port for fHDHR

It gets added as a Source, however without any channels.

I have the ability to 'Scan For Channels' with 2 options:
Broadcast (Over-the-air Antenna)
Cable

Both of those options return the same result:
> Error: Unable to scan Antenna

**Comment by deathbybandaid** [comment]
What origin plugins do you have?

**Comment by lswee** [comment]
I first tried TVHeadend and then IPTV-org, but (as I’ve posted in their respective repositories) neither of those seem to be in working order currently.

Upon further research, this issue with Channels integration seems to be common across “virtual HDHRs”. I’ve tried tvhproxy, xTeVe and Telly - all successfully received content from TVHeadend and IPTV-org, however they exhibited the same error (Unable To Scan Antenna) within Channels.

I got in touch with the developer of Channels and they explained that their app is written to the specs of the HDHR hardware, so the emulators won’t work unless 100% accurate. So, it’s not an impossibility, but I think other apps like Plex are more forgiving in what they’ll accept from virtual HDHR softwares.
